wg-api/spec-documents/scroll-view.md

```diff
@@ -0,0 +1,259 @@
+# Scroll View
+
+## Summary
+
+This spec proposes to add an experimental native ScrollView component to Electron.
+
+## Platforms
+* macOS 
+* Windows
+* Linux
+
+## Impetus
+
+It is currently not possible to add scrolling functionality around BrowserViews:
+
+* Capture mouse / touch events
+* Reposition BrowserViews accordingly
+
+More details of the use-case and challenges related to workarounds: [#32751](https://github.com/electron/electron/issues/32751)
+
+![](https://user-images.githubusercontent.com/2891206/150981006-84fe796e-60bd-43c0-952d-831d1bfd9998.gif)
+
+## Usage Example
+
+```js
+const { BrowserView, BrowserWindow, ContainerView, ScrollView, WrapperBrowserView } = require("electron");
+
+const window = new BrowserWindow({ width: 1000, height: 500 })
```

Comment:
Should this use `BaseWindow` instead of `BrowserWindow`?

Reply:
My current implementation works for both - BaseWindow and BrowserWindow. BrowserWindow has additional features - like triggering the before unload handlers etc. That's why I added to this spec baseWindow.addChild / baseWindow.removeChild (applicable for BrowserWindow too).